pyproject.toml

```diff
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "UMNN"
+version = "1.70"
```

Comment:
Set to `1.70` given https://pypi.org/project/UMNN/1.70/ exists and so assuming that release corresponds to https://github.com/AWehenkel/UMNN/commit/9e489b5fede2206356f803c5bf07064c87ff038a.